Content.Shared/CombatMode/Pacification/PacifiedComponent.cs

```diff
@@ -14,7 +14,7 @@ public sealed partial class PacifiedComponent : Component
     /// <summary>
     ///     A blacklist specifying entities that the player will refuse to throw.
     /// </summary>
-    [DataField("throwBlacklist"), ViewVariables(VVAccess.ReadWrite)]
+    [DataField("throwBlacklist")]
```

Comment:
no leave vv rw alone have `[DataField, ViewVariables(VVAccess.ReadWrite)]` as the full line

datafield is vv readonly by default, its just the tag is optional since the default will be first letter lowercased, throwBlacklist. since thats what you want you dont need to specify the tag

Reply:
*Oh,* I see.